first-task.css

```diff
@@ -203,4 +204,66 @@ li {
 .author-text {
   font-family: 'Plus Jakarta Sans';
   font-size: 13px;
+}
+
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background-color: #232536;
+  color: #FFFFFF;
+  display: flex;
+  justify-content: space-between;
+  padding: 15px 15px;
+  align-items: center;
+}
+.header-home {
+  font-family: 'Poppins';
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 24px;
+}
+.header-blogs{
+  font-family: 'Poppins';
+  font-size: 16px;
+}
+.header-about {
+  font-family: 'Poppins';
+  font-size: 16px;
+}
+.header-contact{
+  font-family: 'Poppins';
+  font-size: 16px;
+}
+.header-button {
+  background-color: #FFFFFF;
+  color: #232536;
+  width: 152px;
+  height: 50px;
```

Comment:
```suggestion
  padding: 15px 40px;
```
😡 Stop setting specific sizes for buttons! They should have dynamic size!
What if I want to put something longer instead of "Subscribe"? The button will remain the same size and it will look broken.
<img width="218" alt="Screenshot 2024-05-23 at 01 34 36" src="https://github.com/Algiannv/First-task-assigned/assets/115032657/e57b4684-8e25-4a20-bccd-3fbe4d387ec5">